packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -108,6 +108,32 @@ module.exports = {
       }
     ],
     loaders: [
+      // Default loader: load all assets that are not handled
+      // by other loaders with the url loader.
+      // Note: This list needs to be updated with every change of extensions
+      // the other loaders match.
+      // E.g., when adding a loader for a new supported file extension,
```

Comment:
I've put this loader first for exactly that reason :-). Maybe add another comment to all other loaders?